modules/apps/sharing/sharing-document-library/src/main/java/com/liferay/sharing/document/library/internal/display/context/SharingDLViewFileVersionDisplayContext.java

```diff
@@ -174,8 +201,27 @@ else if (Objects.equals(
 			i++;
 		}
 
+		DropdownItem sharingDropdownItem =
+			_sharingDropdownItemFactory.createShareDropdownItem(
+				DLFileEntryConstants.getClassName(),
+				_fileEntry.getFileEntryId(), _httpServletRequest);
+
+		UnsafeConsumer<DropdownContextItem, Exception> sharingActionItem =
+			_sharingDropdownItemFactory.createShareActionUnsafeConsumer(
+				DLFileEntryConstants.getClassName(),
+				_fileEntry.getFileEntryId(), _httpServletRequest);
+
 		if (i < dropdownItems.size()) {
```

Comment:
same here than the comment above.

make the FF be the principal if, so this way you can declare only or sharingDropdownItem or sharingActionItem and the removal of the FF is easier.

Reply:
Done!